README.md

```diff
@@ -1,3 +1,13 @@
+
+//intro with description of animal
+//Checkbox confirming your service of free time towards animal
+//Answer of favorite animal "koala
+//3 pictures to choose from for the design
+//Name input allowing to name koala
+![Wireframephoto](Michael Roberts/dev/About-Me/assets)
```

Comment:
you were on the right track here. You can use either relative paths, or full paths. 
A full path would be like a web url, starting with https, and is used when you wanna grab a photo off the internet.
A relative path would be kinda like what you did, but you have to look at where the file is in your directory. What you have there tells the computer: "Look for a folder called 'Michael Roberts'"
That folder doesn't exist inside of where you are currently, so it breaks.
try changing the url to be relative to where the readme is saved in your repo.